README.md

```diff
@@ -1,20 +1,37 @@
 
-This template repository equips you with the default initial files required for a Ministry of Justice GitHub repository.
+This template repository equips you with the default initial files required for an Analytical Platform Image Build Template GitHub repository.
+
+This repository is managed in Terraform [here](https://github.com/ministryofjustice/data-platform/blob/main/terraform/github/analytical-platform-repositories.tf#L286).
 
 ## Included Files
 
 The repository comes with the following preset files:
-
+- .GitHub
+  - workflows
+    - built-and-test.yml
+    - release.yml
+    - super-linter.yml
+    - dependency-review.yml
+- scripts
+  - build-and-test.sh
+- test
+  - container-structure-test.yml
+- Dockerfile
+- .EditorConfig
 - LICENSE
 - .gitignore
 - CODEOWNERS
 - dependabot.yml
 - GitHub Actions example files
 - Ministry of Justice Compliance Badge (public repositories only)
 
+## Features
+
+The base container image is [Ubuntu 22.04 LTS](https://gallery.ecr.aws/ubuntu/ubuntu).
+
```

Comment:
```suggestion
## Features

The base container image is [Ubuntu 22.04 LTS](https://gallery.ecr.aws/ubuntu/ubuntu).

```

We don't want to force someone down a specific OS route, for example in AWS Lambda, we'd use their Amazon Linux 2023 image which is a RHEL type distro.

Reply:
removed this from the README but have not removed from the dockerfile as its core to the template being created , i have added a comment to advise it is interchangeable.